docs/tutorials/access-service-across-clusters.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Access service acrosss clusters within native service
+---
+
+In Karmada, the MultiClusterService can enable users to access services across clusters with the native service domain name, like `foo.svc`, with the aim of providing users with a seamless experience when accessing services across multiple clusters, as if they were operating within a single cluster.
+
+This document provides an example of how to enable MultiClusterService for accessing service across clusters with native service.
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
```

Comment:
Is it necessary to inform users about hack/local-up-karmada.sh is used for e2e cases?

Reply:
I copied from https://karmada.io/docs/next/userguide/scheduling/scheduler-estimator/#karmada-has-been-installed , I think it's ok